addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -730,7 +731,7 @@ export class OdooEditor extends EventTarget {
             if (fontSizeInput && ev.target.closest('#font-size .dropdown-toggle')) {
                 // If the click opened the font size dropdown, select the input content.
                 fontSizeInput.select();
-            } else if (!this.isSelectionInEditable()) {
+            } else if (!this.isSelectionInEditable() && !isColorPickerInput(ev.target)) {
```

Comment:
It's a bit of a red flag to have a reference to a concept that is defined outside of the editor (the colorpicker) within the editor's code. Isn't the problem something more general, like that the target should not be outside the editor (eg, `!ancestors(ev.target).includes(this.editable)`)?

Reply:
I updated the fix with something more general as you requested. I guess it makes sense now , if we click on anywhere in the toolbar and the field is input we don't need to reset latest computed selection.
For your suggested change , it's already in the first part of the condition If I'm not mistaken.

Reply:
> For your suggested change , it's already in the first part of the condition If I'm not mistaken.

Not exactly. The first part of the condition checks that the _selection_ is in the editor. What I was suggesting was checking that the _event's target_ should be in the editor as well (so where the click occurred). But that's wrong anyway because the listener is on the toolbar. What you did seems correct to me.